test/test_distributions.py

```diff
@@ -248,6 +247,58 @@ def test_exponential_sample(self):
                                         scipy.stats.expon(scale=1. / rate),
                                         'Exponential(rate={})'.format(rate))
 
+    def test_laplace(self):
+        mean = Variable(torch.randn(5, 5), requires_grad=True)
+        std = Variable(torch.randn(5, 5).abs(), requires_grad=True)
```

Comment:
nit: Consider renaming this from `std` to `scale`, since this is actually `standard_deviation/sqrt(2)`.